packages/@glimmer/util/lib/platform-utils.ts

```diff
@@ -39,6 +39,8 @@ export type Lit = string | number | boolean | undefined | null | void | {};
 
 export const tuple = <T extends Lit[]>(...args: T) => args;
 
-export const symbol = HAS_NATIVE_SYMBOL
-  ? Symbol
-  : (key: string) => `__${key}${Math.floor(Math.random() * Date.now())}__` as any;
+export function enumerableSymbol(key: string): any {
+  return `__${key}${Math.floor(Math.random() * Date.now())}__`;
+}
+
+export const symbol = HAS_NATIVE_SYMBOL ? Symbol : enumerableSymbol;
```

Comment:
Can you make an issue and leave an inline comment to remove the duplication that this introduces (Ember also has a `enumerableSymbol` and `symbol`)?

Reply:
Issue made: https://github.com/emberjs/ember.js/issues/19295

I'm not going to add a comment here actually since this is a change that needs to happen in Ember, and it will mean we have to make another PR here just to remove a comment about a different codebase